debug_toolbar/panels/templates/panel.py

```diff
@@ -88,7 +88,7 @@ def _store_template_info(self, sender, **kwargs):
         for context_layer in context.dicts:
             if hasattr(context_layer, "items") and context_layer:
                 # Check if the layer is in the cache.
-                key_values = sorted(context_layer.items())
+                key_values = dict(context_layer)
```

Comment:
I think the layers themselves may already be dictionaries anyway – or does it ever happen that this is not the case?

Reply:
Yeah, it seems that way to me in the unit tests and the example app. I've simplified the code with that in mind.